osmaxx/conversion/converters/converter_pbf/to_pbf.py

```diff
@@ -0,0 +1,56 @@
+import os
+import shutil
+import subprocess
+import tempfile
+
+from django.utils import timezone
+
+from rq import get_current_job
+
+from osmaxx.conversion._settings import CONVERSION_SETTINGS, odb_license, copying_notice, creative_commons_license
+from osmaxx.conversion.converters.utils import zip_folders_relative, recursive_getsize
+
+
+def cut_area_from_pbf(pbf_result_file_path, extent_polyfile_path):
+    command = [
+        "osmconvert",
+        "--out-pbf",
+        "--complete-ways",
+        "--complex-ways",
+        "-o={}".format(pbf_result_file_path),
+        "-B={}".format(extent_polyfile_path),
+        "{}".format(CONVERSION_SETTINGS["PBF_PLANET_FILE_PATH"]),
+    ]
+    subprocess.check_call(command)
+
+
+def polyfile_string_to_pbf(polyfile_string, pbf_out_path):
+    polyfile_path = os.path.join('/tmp', 'polyfile_extent.poly')
+    with open(polyfile_path, 'w') as f:
+        f.write(polyfile_string)
+    cut_area_from_pbf(pbf_out_path, polyfile_path)
+
+
+def produce_pbf(*, out_zip_file_path, area_name, polyfile_string):
+    _start_time = timezone.now()
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        out_dir = os.path.join(tmp_dir, 'pbf')
+        os.makedirs(out_dir, exist_ok=True)
```

Comment:
You just used should twice, so you are not certain either? :grin: 
I followed the principle:
Is it an error, if it already exists? No.
Is it possible, that it already exist? Not very likely.
Does it any harm, leaving it in there? It is a bit confusing, at best.
Do we have the same pattern somewhere else? [Yes, and we do the same](https://github.com/geometalab/osmaxx/blob/develop/osmaxx/conversion/converters/converter_garmin/garmin.py#L65)

Result was: Mimik the code we already have, so that refactoring might actually catch that line as well (if we should check for duplicated code, this is more obvious).